### PR TITLE
Feature/bind powerdns

### PR DESCRIPTION
**Pull request by csymons-suse**
change Designate default from PowerDNS to BIND; add DNS overview and configuration sections

### CODE REVIEW

File: xml/operations-dns-designate_initialconfig.xml [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+ <!ENTITY % entities SYSTEM "entities.ent"> %entities;
+]>
+<section xml:id="DesignateInitialConfig"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>&desig; Initial Configuration</title>
+ <para>
+  After the <phrase vendor="hpe">"&hpe; Helion
+  &ostack;</phrase><phrase vendor="suse">&suse; &ostack;</phrase> installation
```
Comment (sknorr) [inline xml/operations-dns-designate_initialconfig.xml]: please use the much simpler `&productname;` for this -- this is always and automatically right and save you a lot of typing.

File: xml/operations-dns-designate_initialconfig.xml [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+ <section>
+  <title>Identify Nameserver Public IPs</title>
+  <para>
+   Depending on the Backend, the method used to identify the nameserver's
+   public IP will differ.
+  </para>
+  <section>
+   <title>InfoBlox</title>
+   <para>
+    InfoBlox will act as your public nameservers, consult the InfoBlox
```
Comment (sknorr) [inline xml/operations-dns-designate_initialconfig.xml]: name server_s_ in the plural? is that correct?

File: xml/operations-dns-designate_initialconfig.xml [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+ <para>
+  After the <phrase vendor="hpe">"&hpe; Helion
+  &ostack;</phrase><phrase vendor="suse">&suse; &ostack;</phrase> installation
+  has been completed, &desig; requires some initial configuration in order to
+  operate.
+ </para>
+ <section>
+  <title>Identify Nameserver Public IPs</title>
+  <para>
+   Depending on the Backend, the method used to identify the nameserver's
```
Comment (sknorr) [inline xml/operations-dns-designate_initialconfig.xml]: don't capitalize "Backend". also, style guide spelling is "back-end".

File: xml/operations-dns-designate_initialconfig.xml [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+]>
+<section xml:id="DesignateInitialConfig"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>&desig; Initial Configuration</title>
+ <para>
+  After the <phrase vendor="hpe">"&hpe; Helion
+  &ostack;</phrase><phrase vendor="suse">&suse; &ostack;</phrase> installation
+  has been completed, &desig; requires some initial configuration in order to
```
Comment (sknorr) [inline xml/operations-dns-designate_initialconfig.xml]: Wordy phrases: Remove "some", remove "in order"

File: xml/installation-installation-designate-install_designate_BIND.xml [hunk 5]
```diff
@@ -10,19 +10,18 @@
  <section>
   <title>Install DNS Service with BIND</title>
   <para>
-   &kw-hos; DNS Service and BIND can be installed together instead of the
-   default <emphasis role="bold">PowerDNS</emphasis> backend. BIND will be
-   deployed to one or more control planes clusters. The following configuration
-   example denotes where the BIND service is installed.
+   &kw-hos; DNS Service defaults to the BIND Backend if another backend is not
```
Comment (sknorr) [inline xml/installation-installation-designate-install_designate_BIND.xml]: don't capitalize "Backend". also, style guide spelling is "back-end".

File: xml/installation-installation-designate-install_designate_BIND.xml [hunk 6]
```diff
@@ -10,19 +10,18 @@
...
+	 configured for domain name service. BIND will be deployed to one or
+   more control planes clusters. The following configuration example denotes
+	 where the BIND service is installed.
```
Comment (sknorr) [inline xml/installation-installation-designate-install_designate_BIND.xml]: Please avoid adding tab characters. (Try running `sed -i 's/\t/ /g' FILE` for a quick fix here.)
There are similar tabbing issues in all the other files touched by this branch.

File: xml/installation-installation-designate-install_designate_InfoBlox.xml [hunk 7]
```diff
@@ -22,12 +22,10 @@
   </para>
  </note>
  <important>
-  <!-- FIXME: hardcoded version numbers, check following para for SOC 8. -
-  sknorr, 2018-01-29 -->
   <para>
-   The InfoBlox integration was enabled and validated in &kw-hos; 3.x. To use
-   InfoBlox with version [4.x / 5.x] we recommend that you review the InfoBlox
-   Deployment guide which can be found here:
+   The InfoBlox integration was enabled and validated in earlier versions. To
```
Comment (sknorr) [inline xml/installation-installation-designate-install_designate_InfoBlox.xml]: This sentence is presumably HPE-only. (I don't know for sure but it very much sounds like it.)

In any case, I think the information in this para can be reduced to "Before you enable and use InfoBlox in \&productname;, make sure to review the InfoBlox Deployment Guide at \<link...\>. Address any further questions with your InfoBlox support contact."

That way, you don't even have to begin bothering with profiling.

File: xml/operations-dns-designate_overview.xml [hunk 8]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entities.ent"> %entities;
+]>
+<section xml:id="DesignateOverview"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>DNS Service Overview</title>
+      <para><phrase vendor="hpe">"&hpe; Helion &ostack;</phrase><phrase vendor="suse">&suse; &ostack;</phrase> DNS Service provides multi-tenant Domain Name Service with REST
```
Comment (sknorr) [inline xml/operations-dns-designate_overview.xml]: Again, `&productname;` please.

File: xml/operations-dns-designate_overview.xml [hunk 9]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
```
Comment (sknorr) [inline xml/operations-dns-designate_overview.xml]: This one looks like it needs to be reformatted... (the other one looks clean though!)

File: xml/installation-installation-designate-install_designate_BIND.xml (hunk 6 above)
Comment (csymons-suse) [reply xml/installation-installation-designate-install_designate_BIND.xml]: strange. I didn't add any tabs. And they look like regular spaces in Atom, which I didn't add either. Do you see them in a text editor or just in the commits here on Github?
All files touched by this branch have had the sed command applied. In both Atom and Kate, they look off. (designate_BIND.xml; line 13 starts with 3 spaces, line 14 2 spaces, line 15 3, line 16 2 spaces).

**Pull request merged**